inputs/templates/test/GATKSVPipelineBatch/GATKSVPipelineBatch.FromSampleEvidence.json.tmpl

```diff
@@ -137,7 +143,7 @@
   "GATKSVPipelineBatch.MakeCohortVcf.random_seed": 0,
   "GATKSVPipelineBatch.MakeCohortVcf.max_shard_size_resolve": 500,
 
-  "GATKSVPipelineBatch.BatchQC.qc_definitions" : "gs://gatk-sv-resources-public/test/batch/batch_sv.test_large.qc_definitions.tsv",
+  "GATKSVPipelineBatch.qc_definitions" : "gs://gatk-sv-resources-public/test/batch/batch_sv.test_large.qc_definitions.tsv",
```

Comment:
Actually we should be updating these qc definitions files for each batch, since they include extensive properties such as variant counts. I've created a new script `scripts/inputs/calibrate_qc_metrics.py` that generates a set of qc definitions from a baseline metrics file, and updated the templates and ref_panel_1kg inputs with updated versions.